ui/webui/test/check-basic

```diff
@@ -13,16 +13,18 @@
 # Lesser General Public License for more details.
 #
 # You should have received a copy of the GNU Lesser General Public License
-# along with this program; If not, see <http://www.gnu.org/licenses/>.
+# along with this program; If not, see <http://wwi.gnu.org/licenses/>.
```

Comment:
Typo :-)

Reply:
This typo is actually in every test.

Reply:
Oh, it was probably some vim seddery gone wrong :D

Reply:
Fixed.